client/src/game.js

```diff
@@ -283,306 +294,337 @@ export class Game {
 
     warnPageReload() {
         return (
+            import.meta.env.PROD &&
```

Comment:
disable the alert when reloading during development, hate closing it every time I save.